firestore-bigquery-export/scripts/import/src/index.ts

```diff
@@ -69,32 +69,33 @@ const validateBatchSize = (value: string) => {
 
 const validateLocation = (value: string) => {
   const index = [
-    "us-west4",
-    "us-west2",
-    "northamerica-northeast1",
-    "us-east4",
-    "us-west1",
-    "us-west3",
-    "southamerica-east1",
-    "us-east1",
-    "europe-west1",
-    "europe-north1",
-    "europe-west3",
-    "europe-west2",
-    "europe-west4",
-    "europe-west4",
-    "europe-west6",
     "asia-east2",
     "asia-southeast2",
     "asia-south1",
     "asia-northeast2",
     "asia-northeast3",
     "asia-southeast1",
-    "australia-southeast1",
     "asia-east1",
     "asia-northeast1",
-    "us",
+    "australia-southeast1",
     "eu",
+    "europe-west1",
```

Comment:
Nit, nit - Any chance you could also order numerically as well like in previous incarnation? i.e. "europe-west1", "europe-west2", etc.

Reply:
This has been added as a separate update in https://github.com/firebase/extensions/pull/914/files

Reply:
@russellwheatley Thanks. Updated!